src/rustllvm/PassWrapper.cpp

```diff
@@ -393,7 +393,8 @@ extern "C" LLVMTargetMachineRef LLVMRustCreateTargetMachine(
     bool TrapUnreachable,
     bool Singlethread,
     bool AsmComments,
-    bool EmitStackSizeSection) {
+    bool EmitStackSizeSection,
+    bool LinkerIsLLD) {
```

Comment:
Maybe call this `RelaxELFRelocations`? This could work with some other linkers too.